ThePensionsRegulator.Frontend/Styles/_tpr-file-upload.scss

```diff
@@ -0,0 +1,43 @@
+﻿
+@import 'govuk/base';
+$button-shadow-size: $govuk-border-width-form-element;
+
+.govuk-file-upload {
+    @include govuk-font-size($size: 19, $line-height: 1);
+    outline: govuk-px-to-rem(3) solid transparent;
+    border-color: transparent;
+    box-shadow: inset 0 0 0 govuk-px-to-rem(2) transparent;
+    padding: govuk-px-to-rem(3) govuk-px-to-rem(3) govuk-px-to-rem(1) govuk-px-to-rem(3);
+    font-family: $govuk-font-family;
+    color: $govuk-text-colour
+}
+
+.govuk-file-upload::file-selector-button {
+    padding-top: (govuk-spacing(2) - $govuk-border-width-form-element);
+    padding-bottom: (govuk-spacing(2) - $govuk-border-width-form-element - ($button-shadow-size / 2));
+    padding-left: $tpr-button-padding;
+    padding-right: $tpr-button-padding *2;
```

Comment:
Your left and right padding should both be `$tpr-button-padding`. You've got one-sided spacing because you've copied values from a button with a chevron, but there is no chevron here.

![image](https://github.com/user-attachments/assets/f41d4232-b9f8-474a-a8c4-cd1e631111f6)

Reply:
yes that's right, I have applies the same style as button but changed now as suggested